atlp-rwanda/atlp-pulse-fn
# Invitation email template

# PR Description

This PR updates the invitation email template.

# Description of tasks that were expected to be completed

- [x] Update the invitation email template to match the required.
- [x] Make sure all the link and redirections uses the correct urls.

# How has this been tested?

1. Update your .env with the correct info
```
ADMIN_EMAIL=***** // the correct admin email you logged in with
ADMIN_PASS=****** // the correct admin password of the above email
REGISTER_FRONTEND_URL=***** // the correct register front-end page url (ex: http://localhost:3001/register) if using localhost on port 3001. the current register page is **/register**
```
2. Login as the admin and navigate to the trainees tab.
3. click the invite user button and insert the email of the trainee to be invited.
4. Lastly, check the email you provide on step 4 for the update template.

# Number of Commits

1

# Please check this Checklist before you submit your PR:
